build.gradle.kts

```diff
@@ -197,6 +196,17 @@ tasks {
         }
     }
 
+    withType(com.github.benmanes.gradle.versions.updates.DependencyUpdatesTask::class) {
+        fun isStable(version: String): Boolean {
+            val stableKeyword = listOf("RELEASE", "FINAL", "GA").any { version.uppercase().contains(it) }
+            val regex = "^[0-9,.v-]+(-r)?$".toRegex()
+            return stableKeyword || regex.matches(version)
+        }
+        gradleReleaseChannel = "current"
+        revision = "release"
+        rejectVersionIf { !isStable(candidate.version) }
+    }
```

Comment:
Do I understand it correctly that we can use that task to update all dependencies if they fulfill the stable criteria? 👍

Reply:
This task just provides a report, but yes, it's the basis for automatic updates, see below.